### PR TITLE
RTD fix

### PR DESCRIPTION
**Pull request by damonge**
Fixes readthedocs issue triggered by new functionality.
I've tested that this runs as it should on readthedocs.

### CODE REVIEW

File: readthedocs/requirements.txt
Comment (nikfilippas) [inline readthedocs/requirements.txt:1]: Are all four necessary? I.e. did  you check that removing any one of those will make RTD fail?

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-22.04
```
Comment (nikfilippas) [inline .readthedocs.yaml:9]: Is there an `ubuntu-latest` option like in GHA?

**Review by nikfilippas — COMMENTED** [review]
I trust that it works, but have a quick look at the comments.

File: readthedocs/requirements.txt
Comment (damonge) [reply readthedocs/requirements.txt:1]: yes, hence the many commits

File: .readthedocs.yaml (hunk 1 above)
Comment (damonge) [reply .readthedocs.yaml:9]: This (and 20) seemed to be the only two options.

**Comment by damonge** [comment]
The generated docs are here https://ccl.readthedocs.io/en/rtd_fix_cached_property/

**Pull request merged**